MekHQ/src/mekhq/campaign/unit/UnitOrder.java

```diff
@@ -219,7 +224,88 @@ public TargetRoll getAllAcquisitionMods() {
             target.addModifier(getCampaign().getCampaignOptions().getIsAcquisitionPenalty(), "Inner Sphere tech");
         }
         // TODO: Fix weight classes
-        // TODO: aero large craft
+        if (entity instanceof Dropship || entity instanceof Jumpship) {
+            //FIXME ?
+            /* In theory, these should be the base target numbers of these vehicles. The base TNs in the book are
+             * static and don't adjust for Admin skill or anything. This doesn't match our usage of skills such as
+             * Administration to find vehicles. Instead, we're going to just kind of assume they 'intended'
+             * a base TN of 7 and adjust for windage.
+             *
+             */
+            if (entity instanceof SpaceStation) {
+                target.addModifier((int) Math.ceil(entity.getCost(false) / 50000000) + 5 - 7, "Spacestation Base");
+            } else if (entity instanceof Warship) {
+                double collars = ((Warship) entity).getDockingCollars().size();
+                target.addModifier((int) (Math.ceil(Math.sqrt(entity.getWeight() / 5000)) +
+                                                Math.ceil(Math.sqrt(collars))) - 7, "Warship Base");
+            } else if (entity instanceof Dropship) {
+                target.addModifier((int) Math.ceil(entity.getCost(false) / 50000000) + 5 - 7, "Dropship Base");
+            } else {
+                int collars = ((Jumpship) entity).getDockingCollars().size();
+                target.addModifier((int) Math.ceil(entity.getCost(false) / 100000000) + collars - 7, "Jumpship Base");
+            }
+
+            // Since the actual rules are ambiguous, I'm using the tech level as a stand-in for 'uniqueness'
+            // 1 is 'common' and 5 is 'experimental' according to enums
+            // However, almost all Dropships are 'D' techlevel, so... a better solution could be found.
+
+            int techLevel = entity.getTechLevel(getCampaign().getGameYear());
+            switch (techLevel) {
+                case 1:
+                    target.addModifier(-1, "Common");
+                    break;
+                case 2:
+                    target.addModifier(0, "Average");
+                    break;
+                case 3:
+                    target.addModifier(3, "Rare");
+                    break;
+                case 4:
+                    target.addModifier(6, "Very Rare");
+                    break;
+                case 5:
+                default:
+                    target.addModifier(10, "Unique");
+
+            }
```

Comment:
Another one that needs discussion, coming up with a solution is fine, but any unnoficial value should really be at least talked about with other devs onto why you are thining of adding it this way, and sometimes also it is worth to bring up those as really valid questions on how to most closely to implement the rules as written on MekHQ, alot of contributors can give great advice and sometimes even have access to erratas on how things changed its workings since release/publication